OSDOCS-8951: Added ROSA with HCP release note (#68936)
openshift/openshift-docs: rosa_release_notes/rosa-release-notes.adoc

[id="rosa-q4-2023_{context}"]
=== Q4 2023

> * **{hcp-title-first} is now available as a cluster creation option.** {hcp-title} provides customers several benefits, focusing on enhanced operational efficiency, increased reliability and resilience, and many architectural benefits that makes {hcp-title} the best place for customers to run their applications in OpenShift on AWS. For more information, see xref:../rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc#rosa-hcp-sts-creating-a-cluster-quickly[Creating ROSA with HCP clusters using the default options].

Reviewer: ```suggestion
* **{hcp-title-first}.** {hcp-title} is now available as a cluster creation option.  {hcp-title} provides customers several benefits, focusing on enhanced operational efficiency, increased reliability and resilience, and many architectural benefits. For more information, see xref:../rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc#rosa-hcp-sts-creating-a-cluster-quickly[Creating ROSA with HCP clusters using the default options].
```

Reviewer: Another option. 
```suggestion
* **Hosted control planes.** {hcp-title-first} is now available as a cluster creation option. {hcp-title} provides customers several benefits, focusing on enhanced operational efficiency, increased reliability and resilience, and many architectural benefits. For more information, see xref:../rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc#rosa-hcp-sts-creating-a-cluster-quickly[Creating ROSA with HCP clusters using the default options].
```
